### ISSUE TITLE
Add some configuration options 

### ISSUE THREAD

**Issue by HuiiBuh** [VasilyShelkov/create-react-extension#26]
First if all. I love your work. I was just amazed how easy the extension was created.

The only thing i am missing is some configuration options regarding the location of the files. 
I would suggest to add an. env file in which you can specify the location of the options/popup/... location.

I had a quick look at thr source code and it looks like you just have to modify the  _packages/react-scripts/config/paths.js_. I would like to create a PR in which some configuration options for the location het added.

**Comment by Dimi-Dun-Morogh** [comment]
yeah I'd love to have an option to exclude contentScript, options folder or w/e is unnecessary for specific extenstion

**Comment by VasilyShelkov** [comment]
Hi, thanks for your comments. I think they're great discussions to have and decide on.

> The only thing i am missing is some configuration options regarding the location of the files.
I would suggest to add an. env file in which you can specify the location of the options/popup/... location.

What's the main reason/use cases why you think it's important to have this configuration? I'm looking to understand if there's something which justifies more configuration for something that isn't personal preference.

The reason I ask is because generally both CRA + this repo want to "standardise" the simple cases. If one wants to build with a different structure, there's no problem with ejecting, however, by having the same structure, it keeps all projects created with `create-react-extension` the same and predictable within the community. Same way that all `create-react-app` projects are within the `src` folder.

**Comment by HuiiBuh** [comment]
The reason I would like to modify the paths is that there are so many entry points in create-react-extension in comparison to create-react-app.
And if I would like to structure these different entry points differently than the default structure I would have to eject which I really don't want to do for such a small change.
I would certainly keep a default configuration which most people could use without even knowing that there are more configuration options available and a more advanced config for people who would like to customize their project a bit more.
This would be possible with an env file and a concept like this is already used by cra if you would like to disable the browser opening on every app start.

**Comment by jonathangaldino** [comment]
I'm glad that I found this issue. I am missing this config file, for the same reasons.
